src/ReactiveUI/ReactiveUI.csproj

```diff
@@ -10,19 +10,22 @@
     <LangVersion>preview</LangVersion>
   </PropertyGroup>
 
-  <PropertyGroup Condition="$(TargetFramework) == 'net5.0-windows'">
+  <PropertyGroup Condition="$(TargetFramework) == 'net5.0-windows' or $(TargetFramework) == 'net6.0-windows'">
     <UseWpf>true</UseWpf>
     <UseWindowsForms>true</UseWindowsForms>
   </PropertyGroup>
 
-  <PropertyGroup Condition=" $(TargetFramework.StartsWith('net46')) or $(TargetFramework) == 'net47' or $(TargetFramework) == 'net471' or $(TargetFramework) == 'net5.0-windows'">
+  <PropertyGroup Condition=" $(TargetFramework.StartsWith('net46')) or $(TargetFramework) == 'net47' or $(TargetFramework) == 'net471' or $(TargetFramework) == 'net5.0-windows' or $(TargetFramework) == 'net6.0-windows'">
```

Comment:
I think we would need separate net 6 conditionals for the different platforms like android and iOS. 

Similar to splat drawing

Reply:
resolved via 0fe0084